src/controllers/menuController.js

```diff
@@ -0,0 +1,72 @@
+const Menu = require("../models/menuModel");
+
+exports.getMenu = async (req, res) => {
+  try {
+    const menu = await Menu.find();
+    res.status(200).json({
+      status: "success",
+      data: menu,
+    });
+  } catch (e) {
+    res.status(400).json({
+      status: "fail",
+      message: e,
+    });
+  }
```

Comment:
```suggestion
  } catch (e) {
  console.log('error', e);
    res.status(500).json({
      message: 'Something went wrong',
    });
  }
```

A couple things here:

1 - It's a good idea to log your errors so you can see whats going on your application logs
2 - Never send out the error in the response to the client, as that can expose application tech stack which is a vulnerability
3 - Error handling is something to learn with experience and some studying, but in this case the status 400 does not apply as the error is not the customer's... 400 is customer error and 500 is application (internal error), and since this catch would happen for no fault of the customer, it's a 500